commands.js

```diff
@@ -23,6 +23,7 @@ const fs = require('fs');
 const MAX_REASON_LENGTH = 300;
 const MUTE_LENGTH = 7 * 60 * 1000;
 const HOURMUTE_LENGTH = 60 * 60 * 1000;
+const BLACKLIST_DURATION = 365 * 24 * 60 * 60 * 1000; // 1 year
 
```

Comment:
For all intensive purposes, I always considered room blacklists to be permanent unless they are manually removed by an actual user.